components/SkillNavigation.styles.js

```diff
@@ -1,17 +1,22 @@
 import css from 'styled-jsx/css';
```

Comment:
p.s. Having the navigation styles duplicated (`SkillNavigation.styles.js` and `RoleNavigation.styles.js`) is yuck.

![alttext](http://66.media.tumblr.com/tumblr_lp1067P4h81qd86gmo2_250.gif)